### PR TITLE
Intro to shipwright - Part 2

### PR DESCRIPTION
**Pull request by adambkaplan**
- Blog post for part 2, introducing the build APIs and install
   instructions.
- Update links for part 1.

### CODE REVIEW

File: content/en/blog/posts/2020-10-21-intro-shipwright-pt1/index.md [hunk 1]
```diff
@@ -8,13 +8,14 @@ resources:
 - src: "**.{png}"
   title: "Figure #:counter"
 ---
+*Update 2020-11-25: Added link to Part 2 of this series*
 
 What is Shipwright? Which problems does this project try to solve?
 
 In [Part 1](/blog/2020/10/15/introducing-shipwright-part-1) of this series, we'll look back at the history of delivering software applications,
```
Comment (zhangtbj) [inline content/en/blog/posts/2020-10-21-intro-shipwright-pt1/index.md:15]: Are these `[Part 1](/blog/2020/10/15/introducing-shipwright-part-1)` and `[Part 2](/blog/2020/11/25/introducing-shipwright-part-2)` a Markdown link from this repo? I found they can not be accessed correctly

File: content/en/blog/posts/2020-11-25-intro-shipwright-pt2.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+1. Source code - the "what" you are trying to build
+2. Output image - "where" you are trying to deliver your application
+3. Build strategy - "how" your application is assembled
+4. Invocation - "when" you want to build your application
+
+Developers who use docker are familiar with this process:
+
+1. Clone source from a git-based repository ("what")
+2. Build the container image ("when" and "how")\
```
Comment (zhangtbj) [inline content/en/blog/posts/2020-11-25-intro-shipwright-pt2.md:49]: an unnecessary `\` at the end of this sentence.

File: content/en/blog/posts/2020-10-21-intro-shipwright-pt1/index.md (hunk 1 above)
Comment (adambkaplan) [reply content/en/blog/posts/2020-10-21-intro-shipwright-pt1/index.md:15]: These are local links in the rendered site. You can test it with the Netlify `deploy-preview` details.

File: content/en/blog/posts/2020-11-25-intro-shipwright-pt2.md (hunk 2 above)
Comment (adambkaplan) [reply content/en/blog/posts/2020-11-25-intro-shipwright-pt2.md]: @zhangtbj fixed in the latest push

**Review by adambkaplan — COMMENTED** [review]
/approve

Invoking owner approval so we can merge on lgtm.

**Review by zhangtbj — APPROVED** [review]

**Pull request merged**